backend/src/main/java/com/gleam/backend/service/GameService.java

```diff
@@ -36,4 +42,40 @@ public Game getGameWithPrefix( String prefix)
                 .min(Comparator.comparing(Game::getTitle, String.CASE_INSENSITIVE_ORDER))
                 .orElse(null);
     }
+
+    public Game getGame(String title) {
+        return getAllGames().stream()
+                .filter(game -> game.getTitle().equalsIgnoreCase(title))
+                .findFirst()
+                .orElse(null);
```

Comment:
Hier sagst du `.orElse(null)` prüfst aber dann in der aufrufenden Methode direkt, ob es null ist. Java bringt leider nicht so besonders gute nullability mit, aber ich würde soweit es geht auf Optionals setzen. Entsprechend würde ich hier auch ein Optional zurückgeben, damit du in der aufrufenden Methode direkt weist, dass hier null drin stecken könnte.